tests/lapi/CMakeLists.txt

```diff
@@ -16,6 +16,13 @@ lapi_tests_make_lua_path(LUA_PATH
   ${CMAKE_CURRENT_SOURCE_DIR}/?.lua
 )
 
+list(APPEND TEST_ENV
+  "LUA_PATH=${LUA_PATH};"
```

Comment:
Minor: I suppose there is no need for `;` here and below, since it is already the list.